Sentry/src/test/java/io/sentry/NoOpTransactionProfilerTest.kt

```diff
@@ -0,0 +1,15 @@
+package io.sentry
+
+import kotlin.test.Test
+import kotlin.test.assertNull
+
+class NoOpTransactionProfilerTest {
+    private var profiler: NoOpTransactionProfiler = NoOpTransactionProfiler.getInstance()
+
+    @Test
+    fun `onTransactionStart is no op`() = profiler.onTransactionStart(NoOpTransaction.getInstance())
```

Comment:
There's no assert here, What does the test do?

Reply:
yeah, I copied it from other NoOp...Test classes
Changed it to `assertEquals(Unit, profiler.onTransactionStart(NoOpTransaction.getInstance()))`

Reply:
Maybe we rename to `methodX does not throw`, since we cannot assert anything to tell its `no op` or not, right? unless you can.

Reply:
yeah, i cannot assert anything, so will do that `methodX does not throw`